src/wms/src/main/java/org/geoserver/wms/WMSXStreamLoader.java

```diff
@@ -63,7 +63,10 @@ public static void initXStreamPersister(XStreamPersister xp) {
         xs.alias("wms", WMSInfo.class, WMSInfoImpl.class);
         xs.registerConverter(new WMSInfoConverter(xp));
         xs.addDefaultImplementation(WatermarkInfoImpl.class, WatermarkInfo.class);
```

Comment:
Unsure here, but how does the generated look like, does it conform to the rest, or do we need to add an alias for CacheConfiguration? While we should not worry too much about the disk storage, the config is also visible in the REST API, both XML and JSON representations.

Reply:
in the storage it looks like this:
```xml
<cacheConfiguration>
    <enabled>false</enabled>
    <maxEntries>1000</maxEntries>
    <maxEntrySize>51200</maxEntrySize>
</cacheConfiguration>
```
Do you think it should work also for the others?

Reply:
Yep, looks good to me